shap/explainers/_sampling.py

```diff
@@ -12,7 +12,8 @@
 
 
 class Sampling(Kernel):
-    """ This is an extension of the Shapley sampling values explanation method (aka. IME)
+    """Computes SHAP values using an extension of the Shapley sampling values explanation method
+    (also known as IME).
```

Comment:
As a nitpick, to fully adhere to google-style docstring convention the first line summary should fit on one line only. I don't think small deviations matter particularly though, as the Sphinx theme still displays ok with slightly longer summaries.

Reply:
I'm not particularly bothered by it in this case, since it's only "long" because of the name of the method itself, not because we're cramming a lot of information into 1 sentence (i.e., conceptually, it's a short sentence).

Will resolve this as is, but thanks for raising this in the review regardless!